### ISSUE TITLE
Invalid token

### ISSUE THREAD

**Issue by vgdev-prog** [codetime-dev/codetime-web#16]
When I try to insert code from the site into the plugin, phpstorm gives an error
![codetime](https://github.com/user-attachments/assets/bf7732ae-7811-4e51-908c-4338a49fbdfd)

**Comment by Jannchie** [comment]
Hi, thanks for using Code Time. I noticed you downloaded an older version of the plugin that has been deprecated. You can find the new version of the plugin here: https://plugins.jetbrains.com/plugin/25617-codetime

**Comment by Tienam123** [comment]
![Screenshot from 2024-12-14 18-00-01](https://github.com/user-attachments/assets/7b7db9af-6d60-4c40-8a95-11b91ce28ecf)

with the new plugin the situation is similar for jetbrains b vscode

**Comment by Jannchie** [comment]
Strange. May I ask if you are using https://codetime.dev? Because there might be other services with the same name…

**Comment by Tienam123** [comment]
yes , and refresh token don't accept that couple of days i have that problem

**Comment by Jannchie** [comment]
I apologize for any inconvenience caused. 

However, I haven’t been able to pinpoint where the issue might be...

I may need you to provide more information. Could you please try using the command below to check the output? If everything is working correctly, you should receive a JSON response.

```
curl -H "token: <your-token-here>" "https://api.codetime.dev/user/minutes?minutes="
```

**Comment by Tienam123** [comment]
![Screenshot from 2024-12-14 21-17-17](https://github.com/user-attachments/assets/3ab7d34c-e228-41d2-9ba4-7edceac522ae)

it tells me there is no such user) although before copying I updated the token in the personal account

**Comment by Tienam123** [comment]
and next request with new updated token response with user not found

**Comment by Jannchie** [comment]
I see. This might be a frontend issue, possibly related to the browser or operating system. I noticed that you seem to be using Linux, though I’m not sure which distribution it is. I’ll try to fix this issue.

**Comment by Tienam123** [comment]
Yes I use ubuntu 24.04 but I had the same problem when I tried to do it on windows from my laptop

**Comment by Jannchie** [comment]
May I ask what the difference is between the token content copied to the clipboard from the webpage and the response payload?

**Comment by Tienam123** [comment]
I checked both of these tokens and they turned out to be absolutely identical, but I decided to test again and first I pasted the code copied from the front end into vs code and the error repeated itself, then I repeated the operation again and copied it from the API and everything worked.
This is screen from VS Code when i paste front-end token
![Screenshot from 2024-12-16 07-15-06](https://github.com/user-attachments/assets/6dca9b7d-307e-4f58-9c31-e964b21c166d)

Then i ckecked both of them in js and see both identical
![Screenshot from 2024-12-16 07-22-21](https://github.com/user-attachments/assets/49301847-3bb4-4b18-8dce-42877fa2beb3)

Maybe this can help you

**Comment by siaikin** [comment]
@Jannchie Hi 我注意到webstorm插件市场里有两个名为 codetime 的插件，https://plugins.jetbrains.com/plugin/15507-codetime 和 https://plugins.jetbrains.com/plugin/25617-codetime。在此之前我一直使用前者，直到我遇见 “Invalid token”。

如果前者已弃用，请在网站中提示。

**Comment by QuarkPixel** [comment]
I also have this problem
![image](https://github.com/user-attachments/assets/2e83df75-2b74-4eb3-aff7-dc622c7632f9)

**Comment by Jannchie** [comment]
The plugin in the screenshots (plugins.jetbrains.com/plugin/15507) is the deprecated one; the current plugin is https://plugins.jetbrains.com/plugin/25617-codetime, and that is the only one the dashboard links to now. The token panel has also been rebuilt since this report.

Closing as stale — please comment or open a new issue if `invalid token` still happens with the current plugin.
